### PR TITLE
Prevent $char->{party} autovivification when checking whether we're in a party

### LINKED ISSUE #1408: party create error
iRO: Restart
Problem on creating party using kore, 

dafuq

party request works when you are already a party leader. Need fix. Thx

### PR DESCRIPTION
**Pull request by lututui**
Might fix #1408

### CODE REVIEW

File: src/Network/Receive/kRO/Sakexe_0.pm [hunk 1]
```diff
@@ -2069,11 +2069,12 @@ sub party_join {
 	$name = bytesToString($name);
 	$user = bytesToString($user);
 
-	if (!$char->{party} || !%{$char->{party}} || !$char->{party}{users}{$ID} || !%{$char->{party}{users}{$ID}}) {
+	if (!$char->{party}{joined} || !$char->{party}{users}{$ID} || !%{$char->{party}{users}{$ID}}) {
```
Comment (sctnightcore) [inline src/Network/Receive/kRO/Sakexe_0.pm]: ty XD

**Comment by lututui** [comment]
Wait for @fadreus or @adraaawr feedback before merging

**Comment by alisonrag** [comment]
@lututui 
the merge is with you

**Comment by adraaawr** [comment]
@lututui this fixed #1408 thanks!

**Comment by alisonrag** [comment]
@lututui
Tested you can merge now

**Pull request merged**